scripts/lua/get_host_pools.lua

```diff
@@ -14,30 +14,65 @@ sendHTTPHeader('text/html; charset=iso-8859-1')
 local ifid = _GET["ifid"]
 local pool_id = _GET["pool"]
 local res = {data={}, sort={{"column_", "asc"}}, totalRows=0}
+local currpage = tonumber(_GET["currentPage"]) or 1
+local perpage = tonumber(_GET["perPage"]) or 5
+
+local start_i = (currpage-1) * perpage
+local stop_i = start_i + perpage - 1
+local i = 0
 
 if((ifid ~= nil) and (isAdministrator())) then
   if pool_id ~= nil then
+    local active_hosts = interface.getHostsInfo(false, nil, nil, nil, nil, nil, nil, nil, nil, nil, true--[[no macs]], tonumber(pool_id)).hosts
+    local network_stats = interface.getNetworksStats()
+
     for _,member in ipairs(host_pools_utils.getPoolMembers(ifid, pool_id)) do
-      res.data[#res.data + 1] = {
-        column_member = member.address,
-        column_vlan = member.vlan,
-      }
+      if (i >= start_i) and (i <= stop_i) then
+        local _, key = getRedisHostKey(member.key)
+        local link
+
+        if active_hosts[key] then
```

Comment:
use helper functions (eg., isIPv or isMac) to avoid doing unnecessary interface. call